.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```

Comment:
These are IDE-specific files and you may ignore them or even delete them. I'll make a new repo to include a `.gitignore` file so IDE-related files won't be committed.